config/calibrateImage.py

```diff
@@ -0,0 +1,36 @@
+# DRP CONFIGURATION FOR CalibrateImageTask
+#
+# This configuration can't go in the task defaults, because those are shared
+# with AP.  It could be moved directly into a pipeline YAML after the v1->v2
+# migration is done.
+
+# In DRP not calibrating the pixels (or the backgrounds) until we have
+# our final photometric calibration makes everything downstream much
+# simpler.
+do_calibrate_pixels: False
```

Comment:
Is there a missing `config.` here?

Reply:
Ah, yes, good catch!

Especially since - because I used this file to avoid duplication between the new and old pipelines - this is a rare change on this ticket that would have broken more than just the new pipeline!